dulwich/tests/test_object_store.py

```diff
@@ -243,6 +244,59 @@ def test_iter_tree_contents_include_trees(self) -> None:
         actual = iter_tree_contents(self.store, tree_id, include_trees=True)
         self.assertEqual(expected, list(actual))
 
+    def test_iter_commit_contents(self) -> None:
```

Comment:
Please add a separate TestCase subclass for this.

Reply:
Done